content/index.ipynb

```diff
@@ -168,7 +168,7 @@
     "## 2.5 Statistical analysis\n",
     "\n",
     "\n",
-    "Statistical analysis was conducted using the SciPy Python library `v1.10.1` {cite:p}`Virtanen2020-hl`. Descriptive statistics, including mean and standard deviation, were computed for age, height, and weight. The Shapiro-Wilk normality test was used to assess data normality. Differences between males and females in age, height, and weight were examined using the Wilcoxon rank-sum test. Morphometric measures in PAM50 space were averaged across participants for each slice and compared between sex and MRI vendors using the Wilcoxon rank-sum test. The significance level was set to `alpha = 0.001`.\n",
+    "Statistical analysis was conducted using the SciPy Python library `v1.10.1` {cite:p}`Virtanen2020-hl`. Descriptive statistics, including mean and standard deviation, were computed for age, sex, height, and weight. The Shapiro-Wilk normality test was used to assess data normality. Differences between males and females in age, height, and weight were examined using the Wilcoxon rank-sum test. Morphometric measures in PAM50 space were averaged across participants for each slice and compared between sex and MRI vendors using the Wilcoxon rank-sum test. The significance level was set to `alpha = 0.001`.\n",
```

Comment:
For sex, we report only the subject count. Not mean and std.
<img width="857" alt="image" src="https://github.com/valosekj/PAM50-normalized-metrics-paper/assets/39456460/65712dab-9d18-4efe-8369-f3843447ce16">

Reply:
`sex` removed in https://github.com/valosekj/PAM50-normalized-metrics-paper/pull/4/commits/7ef49d20ff28111709a7c4ad1ea15a04050d29c8